aws/cf_template.go

```diff
@@ -346,3 +351,26 @@ func generateTemplate(spec *stackSpec) (string, error) {
 
 	return string(stackTemplate), nil
 }
+
+func denyInternalTraffic(template *cloudformation.Template, listenerName string) {
+	template.AddResource("HTTPRuleBlockInternalTraffic", &cloudformation.ElasticLoadBalancingV2ListenerRule{
```

Comment:
Can't be called twice, one of the resources would overwrite the other one.

Reply:
Yes, I still have to add test for that too. That's a draft with the code I used for a POC in order to load test. It shouldn't be reviewed.